src/orion/algo/pbt/pbt.py

```diff
@@ -384,14 +402,15 @@ def _fork_lineages(self, num):
 
             logger.debug("Forking trial %s to %s", trial_to_branch, new_trial)
 
+            assert new_trial is not None
```

Comment:
Another assert

Reply:
See other reply, but indeed this one is not needed (since the one above is sufficient)

Reply:
Fixed in 168d99384cc8b8d2cb8c35c7097599d8b03c0783